blocks/identity-block/components/Identity.jsx

```diff
@@ -1,26 +1,41 @@
-// import { useState } from 'react';
+import { useState } from 'react';
 import Identity from '@arc-publishing/sdk-identity';
-// import { useFusionContext } from 'fusion:context';
-// import getProperties from 'fusion:properties';
-// import { isServerSide } from '@wpmedia/engine-theme-sdk';
+import { useFusionContext } from 'fusion:context';
+import getProperties from 'fusion:properties';
+import { isServerSide } from '@wpmedia/engine-theme-sdk';
 
-const useIdentity = () => ({
-  Identity,
-  isInitialized: true,
-});
+const useIdentity = () => {
+  const { arcSite } = useFusionContext();
+  const { subscriptions } = getProperties(arcSite);
```

Comment:
Because I thought it would be more future proof, the latest `Fusion-News-Theme` `blocks.json` has this in the format of `const { api } = getProperties(arcSite)` using the hierarchy `api.identity.origin`

This will impact the lines below, but I will note those individually.

Reply:
@nschubach thanks, that's a great suggestion -- this should be future proof us for commerce use cases